lib/DAV/GroupFoldersHome.php

```diff
@@ -77,7 +77,7 @@ private function getFolder(string $name): ?array {
 	private function getDirectoryForFolder(array $folder): GroupFolderNode {
 		$userHome = '/' . $this->user->getUID() . '/files';
 		$node = $this->rootFolder->get($userHome . '/' . $folder['mount_point']);
-		return new GroupFolderNode(Filesystem::getView($userHome), $node, $folder['folder_id']);
+		return new GroupFolderNode(Filesystem::getView(), $node, $folder['folder_id']);
```

Comment:
Hum ugly, looks like original intent was something like new View($userHome), but changing this now will have effects, should double check what should be done.
If $userHome is not used anymore it shoudl also be removed.

Reply:
The logic of getView() already returns a View for the user home by default if a user exists, so nothing changes. $userHome is still used to construct the $node so that's why it is not removed.